pkg/restapi/vc/operation/operations.go

```diff
@@ -1690,7 +1690,7 @@ func (o *Operation) parseAndVerifyVP(vpBytes []byte) (*verifiable.Presentation,
 			return nil, err
 		}
 		// verify if the credential in vp is valid
-		_, err = o.parseAndVerifyVCStrictMode(vcBytes)
+		err = o.validateCredentialProof(vcBytes, nil)
```

Comment:
saw this `// TODO figure out the process when vc has more than one proof`, I think we should verify all.

Reply:
Yes, we need to implement that.

Reply:
#412